db/migrate/20170120164058_create_task_for_each_job_and_transfer_attributes.rb

```diff
@@ -0,0 +1,29 @@
+class CreateTaskForEachJobAndTransferAttributes < ActiveRecord::Migration[5.0]
+  class Job < ActiveRecord::Base
+    belongs_to :miq_task
+    self.inheritance_column = :_type_disabled
+  end
+
+  class MiqTask < ActiveRecord::Base
+    has_one :job
+  end
+
+  def up
+    say_with_time("Creating tasks associated with jobs") do
+      Job.find_each do |job|
+        job.create_miq_task(:status        => job.status.try(:capitalize), :name  => job.name, :message => job.message,
+                            :state         => job.state.try(:capitalize), :userid => job.userid,
+                            :miq_server_id => job.miq_server_id, :context_data => job.context,
+                            :started_on    => job.started_on)
+      end
+    end
+  end
+
+  def down
+    say_with_time("Deleting all tasks which have job") do
+      Job.find_each do |job|
```

Comment:
This can be scoped to only jobs that have and `miq_task_id` so that you don't have to load every job.